extension/src/popup/locales/en/translation.json

```diff
@@ -200,6 +201,10 @@
   "Got it": "Got it",
   "Hash": "Hash",
   "Help": "Help",
+  "Hide payments smaller than 0": {
+    "1 XLM": "Hide payments smaller than 0.1 XLM"
+  },
+  "Hide small payments": "Hide small payments",
```

Comment:
Not 100% sure why this translation strings weren't already committed, but no problem adding them now